mithril-aggregator/src/database/repository/cardano_transaction_repository.rs

```diff
@@ -193,6 +207,47 @@ mod tests {
         );
     }
 
+    #[tokio::test]
+    async fn repository_get_transaction_by_hashes() {
+        let connection = Arc::new(cardano_tx_db_connection().unwrap());
+        let repository = CardanoTransactionRepository::new(connection);
+        repository
+            .create_transaction("tx-hash-123", 10, 50, "block_hash-123", 99)
```

Comment:
We may use create_transactions to have .await.unwrap() only one time like in the test `repository_store_transactions_and_get_stored_transactions`